### DISCUSSION TITLE
Alternative to netcdf

### DISCUSSION THREAD

**Discussion by jguterl** [ORNL-Fusion/GITR#70] [General]
Idea: under the assumption that pre and post processing tools will be in python, why not read and write directly the data structure into numpy format (e.g. https://github.com/rogersce/cnpy).
That should be added on top of netcdf capabilities for backward compatibility by analyzing extension of files and as an option for output format. One could also get rid of netcdf at compilation time if required by the user (will simplify the compilation)

**Comment by stonecoldhughes** [comment]
Possible alternatives list:

https://bout-dev.readthedocs.io/en/latest/user_docs/bout_options.html
